GeometryCore/include/LinearAlgebra.h

```diff
@@ -31,37 +32,46 @@ namespace geom_utils
     template<typename T>
     constexpr typename Point3D<T>::coord dot(const Point3D<T>& p, const Point3D<T>& p2)
     {
-        static_assert(false, "dot() is not implemented yet for Point3D");
-        return {};
+        return p.x * p2.x + p.y * p2.y + p.z * p2.z;
     }
 
     template<typename T>
     constexpr typename Point3D<T>::coord cross(const Point3D<T>& p, const Point3D<T>& p2)
     {
-        static_assert(false, "cross() is not implemented yet for Point3D");
-        return {};
+        return p.y * p2.z - p.z * p2.y + p.z * p2.x - p.x * p2.z + p.x * p2.y - p.y * p2.x;
     }
```

Comment:
Cross product for two 3D vectors is another **3D vector**, which is ortogonal to input vectors. Result is not scalar!

Reply:
Ok, i will implement this